playbooks/common/openshift-cluster/reset_excluder.yml

```diff
@@ -5,4 +5,12 @@
   tasks:
   - include_role:
       name: openshift_excluder
+      tasks_from: init
+  - include_role:
+      name: openshift_excluder
       tasks_from: reset
```

Comment:
Similar comment here, maybe an `finish_upgrade.yml` task file that does init/reset?